Bk

macsilber opened the pull request.

DarrenCook committed:
Initial datasets

DarrenCook committed:
Updated readme

DarrenCook committed:
Added in the book code

DarrenCook committed:
markdown format fix

DarrenCook commented:
The idea was to keep all content in the bk branch (we have a few direct URLs in the book, and this kept them short enough to fit!); and therefore (to avoid any confusion) have nothing but the readme in the master branch. But let me know if that is causing a problem.  (Was there anything else in this commit, other than copying "bk" to "master"?)

DarrenCook committed:
Inconsistent variable naming, and one key line was missing. Also added some comments.

DarrenCook committed:
Added the php script used to organize the football data

DarrenCook committed:
Added missing movie.tfidf.csv

duncanmg committed:
Script wouldn't run because of undefined variable:

Warning message:
In h2o.summary(data$class) :
  Approximated quantiles computed! If you are interested in exact quantiles, please pass the `exact_quantiles=TRUE` parameter.
Error in is.H2OFrame(fr) : object 'd' not found
Calls: h2o.splitFrame -> chk.H2OFrame -> is.H2OFrame

Changed 'd' to 'data', runs now and the data looks plausible (when the path is correct).

DarrenCook committed:
Merge pull request #6 from duncanmg/undefined_variable

Script wouldn't run because of undefined variable:

DarrenCook committed:
Removed junk (duplicate) code

DarrenCook commented:
Pending some problem  (with keeping all code and datasets in "bk" branch), I'm closing this.
